assembly/std/solidity_compat/bytes.ts

```diff
@@ -0,0 +1,103 @@
+import { Serializable, stringToBytes } from '@massalabs/as-types';
```

Comment:
```suggestion
import { Args, Result, Serializable, stringToBytes } from '@massalabs/as-types';
```

Reply:
Removed section so no need to pull in Args & Result